flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/sink/TestFileStore.java

```diff
@@ -55,13 +55,15 @@
 
     public boolean expired = false;
 
+    public boolean hasPk;
```

Comment:
Add a constructor: `public TestFileStore(boolean withPrimaryKey)`?

Reply:
> TestFileStore

Sure. Then we cannot resue `fileStore`